README.md

```diff
@@ -70,6 +70,7 @@ A curated list of awesome Java frameworks, libraries and software. Inspired by o
 *Tools that provide metrics and quality measurements.*
 
 * [Checkstyle](http://checkstyle.sourceforge.net/) - Static analysis of coding conventions and standards.
+* [ErrorProne](https://github.com/google/error-prone) - Catch common Java mistakes as compile-time errors from Google.
```

Comment:
You misunderstood the issue here. The preposition *from* indicates that these errors are a result of some code Google wrote. Please fix the wording and the spelling of the library.